proto/cellar.proto

```diff
@@ -0,0 +1,230 @@
+/*
+* An API for cellars that implement the EIP4626-compatible Cellar.sol contract.
+*
+* See [Cellar.sol](https://github.com/PeggyJV/cellar-contracts/blob/main/src/base/Cellar.sol)
+*/
+
+syntax = "proto3";
+package steward.v2;
+
+option go_package = "/steward_proto";
+
+/*
+* Represents a function call to a cellar that implements Cellar.sol
+*/
+message Cellar {
```

Comment:
Given that we have a Cellar V2 approaching in the near term, should we version this?

Reply:
Yeah, just figured maybe `CellarV1` or something would be more appropriate so that there isn't an "unversioned" cellar name, but not critical.